lib/search.js

```diff
@@ -34,11 +45,62 @@ function initialRequest (opts) {
 function extaractDeveloperId (link) {
   return link.split('?id=')[1];
 }
+// Mappings used for normal search results (no misspellings aka no suggested results)
+// change these mappings if searches not producing results and term is causing suggested results to be served
+let INITIAL_MAPPINGS = {
+  app: ['ds:4', 0, 1, 0, 23],
+  sections: ['ds:4', 0, 1]
+};
+let SECTIONS_MAPPING = {
+  title: [22, 1, 0],
+  token: [22, 1, 3, 1],
+  apps: [22, 0],
+  noResults: [25, 0, 0, 0, 1],
+  suggestedResultDescription: [25, 0, 0, 1, 1],
+  aboutResultsTitle: ['ds:4', 0, 1, 0, 31, 0]
+};
+
+let SUGGESTED_APPS_MAPPING = [];
+let SUGGESTED_APPS_EU_MAPPING = [];
 
 async function processFirstPage (html, opts, savedApps, mappings) {
   if (R.is(String, html)) {
     html = scriptData.parse(html);
   }
+  // For US country code searches, if suggested results are given the mappings will be different
+  // change these mappings to fix any suggest search issues related to US country code
+  if (opts.country === 'US' && typeof R.path(['ds:4', 0, 1, 0, 25, 0, 1, 1, 1], html) === 'string') {
```

Comment:
can you update this to make it independent of specific country? we don't know for what subset of countries the different layout is returned, it could change in the future even if now it's us only (or US could be changed to match the rest). If there are two different cases ideally the code should figure out which one to use by inspecting the data.